Lib/fontTools/misc/xmlReader.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, fileOrPath, ttFont, progress=None, quiet=None):
 			self.quiet = quiet
 		self.root = None
 		self.contentStack = []
+		self.contentOnly = contentOnly
```

Comment:
Did you notice `XMLReader.read(..., rootless=True)`?  That should do the same, no?

Reply:
It's not really the same.  

From reviewing the code, I gather that rootless option allows for skipping the <ttFont> root tag along with the version reading.  The contentOnly is intended to skip the parsing of the table tag and instead allow parsing of a table's content  from a subfile directly into an in-progress table.  This fits the situation we have with splitting a <glyf> table across many subfiles.  Each of the subfiles will need to parse it's content and contribute the content to the in-progress <glyf> table.  This situation is currently detected when the xmlReader encounters a level-2 node which has a src.  It knows that it's already inside of a table and is being asked to parse the table's content from a subfile.  This is contrasted to when it reads an entire table from a subfile when it is at a level-1 node and detects a src attribute.

An example of the output needing to be parsed is below:

Example <Fabrikat-Black._g_l_y_f> fil structure:

``` 
<?xml version="1.0" encoding="UTF-8"?>
<ttFont ttLibVersion="3.15">
  <glyf>
    <!-- The xMin, yMin, xMax and yMax values
         will be recalculated by the compiler. -->
    <TTGlyph src="Fabrikat-Black._g_l_y_f._2e_n_o_t_d_e_f.ttx"/>
    <TTGlyph name=".null"/><!-- contains no outline data -->
    <TTGlyph ...>
  </glyf>
</ttFont>

```

Then, an example of one of the files which would need to be parsed with contentOnly:
 <Fabrikat-Black._g_l_y_f._2e_n_o_t_d_e_f.ttx>

```
<?xml version="1.0" encoding="UTF-8"?>
<ttFont ttLibVersion="3.15">

  <TTGlyph name=".notdef" xMin="65" yMin="-60" xMax="495" yMax="770">
    <contour>
      <pt x="65" y="770" on="1"/>
      <pt x="95" y="770" on="1"/>
      <pt x="95" y="-60" on="1"/>
      <pt x="65" y="-60" on="1"/>
    </contour>
   <contour.../>
  </TTGlyph>
</ttFont>

```